packages/gatsby-plugin-gatsby-cloud/src/components/buttons/InfoIndicatorButton.js

```diff
@@ -218,37 +221,91 @@ const InfoIndicatorButton = ({
           onClick: onTooltipToogle,
         })
       },
-      [BuildStatus.BUILDING]: () => {
-        setButtonProps({
-          ...initialButtonProps,
-          tooltip: {
-            testId: initialButtonProps.testId,
-            content: `Building a new preview`,
-            overrideShow: true,
-            hoverable: false,
-          },
-          active: true,
-          hoverable: false,
-          showSpinner: true,
-        })
-      },
     }
 
-    const buildStatusAction = buildStatusActions[buildStatus]
+    // this is because the build status enum is used for ui state - so we can't have 1 ui state that covers multiple build statuses.
+    // If we don't have content sync info (pollForNodeManifest), we have to assume that a currently building build isn't applicable to the current user. So we default to the least noisy state which is UPTODATE.
+    // @todo refactor and decouple build states from UI states - this state should be something like uiState === `IDLE`
+    buildStatusActions[BuildStatus.BUILDING] =
+      buildStatusActions[BuildStatus.UPTODATE]
+
+    const contentSyncSuccessAction = contentSyncRedirectUrl
+      ? () => {
```

Comment:
we need to mimic `[BuildStatus.SUCCESS]: () => {` here more, there is some click functionality that is missing